### PR TITLE
added yarascan output to HTML report

### PR DESCRIPTION
**Pull request by superponible**

### CODE REVIEW

**Comment by botherder** [comment]
This should remain on hold for #564

**Comment by jbremer** [comment]
Looks like it would work - shall we just merge it? Whether we're getting rid of HTML later on is not really relevant for this PR.

File: data/html/sections/volatility.html [hunk 1]
```diff
@@ -64,6 +64,38 @@
...
+                        <tr>
+                            <th>Rule</th>
+                            <th>Owner</th>
+                            <th style="width: 60%">Hexdump</th>
+                        </tr>
+                        {% for match in results.memory.yarascan.data %}
+                            <tr>
+                                <td>{{match.rule}}</td>
+                                <td>{{match.owner}}</td>
+                                <td><span class="mono"><pre>{match.hexdump}}</pre></span></td>
```
Comment (jbremer) [inline data/html/sections/volatility.html]: Looks like a typo.

**Comment by jbremer** [comment]
Closing this PR as we're replacing the old HTML report with a new one which will be including the Yara matches.

**Pull request closed without merging**